.github/workflows/docker_images.yml

```
@@ -778,7 +778,7 @@ jobs:
           docker cp docs/notebook_validation.py cuda-quantum:"/home/cudaq/notebook_validation.py"
           # In containers without GPU support, UCX does not work properly since it is configured to work with GPU-support.
           # Hence, don't enforce UCX when running these tests.
-          docker exec cuda-quantum bash -c "python3 -m pip install pandas scipy pandas seaborn"
+          docker exec cuda-quantum bash -c "python3 -m pip install pandas scipy pandas seaborn h5py==3.10.0"
```

We have a few other places that have h5py restrictions (like devdeps.Dockerfile and pyproject.toml). It would probably be best to specify it in the same way as them.
```suggestion
          docker exec cuda-quantum bash -c "python3 -m pip install pandas scipy pandas seaborn h5py<3.11"
```